example/docker-compose/etc/otel-collector.yaml

```diff
@@ -0,0 +1,15 @@
+receivers:    
+  jaeger:     
+    protocols:
+      thrift_http:
+exporters:
+  otlp:
+    endpoint: tempo:55680
+    insecure: true
+    headers:
+      x-scope-orgid: authed-tenant
```

Comment:
Is there a value that might be clearer than `authed-tenant` that it is a placeholder for a real value?  Thinking something like `foo` or `123`